Documentation/install/aws/aws-terraform.md

```diff
@@ -10,7 +10,6 @@ Generally, the AWS platform templates adhere to the standards defined by the pro
 * **Terraform**: Tectonic Installer includes and requires a specific version of Terraform. This is included in the Tectonic Installer tarball. See the [Tectonic Installer release notes][release-notes] for information about which Terraform versions are compatible.
 * **DNS**: Ensure that the DNS zone is already created and available in Route 53 for the account. For example if the `tectonic_base_domain` is set to `kube.example.com` a Route 53 zone must exist for this domain and the AWS nameservers must be configured for the domain.
```

Comment:
Entries created in this zone are expected to be resolvable from the nodes. In most cases this means that the zone that you are configuring must be a public resolvable zone. You can verify this by using dig to determine the nameserver of this zone. 

dig NS kube.example.com @8.8.8.8 

If you see NS servers that match what you have configured in route53. Great!